repair/row_level.cc

```diff
@@ -2429,6 +2435,52 @@ future<> repair_service::init_ms_handlers() {
     ms.register_repair_get_diff_algorithms([] (const rpc::client_info& cinfo) {
         return make_ready_future<std::vector<row_level_diff_detect_algorithm>>(suportted_diff_detect_algorithms());
     });
+    ms.register_repair_update_system_table([this] (const rpc::client_info& cinfo, repair_update_system_table_request req) -> future<repair_update_system_table_response> {
+        auto from = cinfo.retrieve_auxiliary<gms::inet_address>("baddr");
```

Comment:
It's nicer if this is in a named function so stack traces don't should this as init_ms_handlers()::lambda#17():...

Reply:
OK.